client/src/advertising/with_us/index.tsx

```diff
@@ -51,46 +51,89 @@ export function AdvertiseWithUs() {
         </section>
       </div>
       <section>
-        <h2>Our audience</h2>
+        <h2>Join MDN's Privacy-First Ad Journey</h2>
         <p>
-          MDN is an open source documentation resource for web developers. It
-          primarily consists of reference and educational documentation on a
-          wide variety of subjects, including{" "}
-          <a href="/en-US/docs/Web/CSS">CSS</a>,{" "}
-          <a href="/en-US/docs/Web/HTML">HTML</a>,{" "}
-          <a href="/en-US/docs/Web/JavaScript">JavaScript</a>, and{" "}
-          <a href="/en-US/docs/Web/API/">Web APIs</a>. We also provide an
-          extensive set of <a href="/en-US/docs/Learn">learning resources</a>{" "}
-          for new developers and students, accounting for approximately 10% of
-          our audience. More than 17 million web developers visit our website
-          every month, and we pride ourselves with a customer satisfaction of
-          over 90%.
+          At MDN, we champion user privacy and we stand by{" "}
+          <a
+            href="https://www.mozilla.org/en-US/privacy/ad-targeting-guidelines/"
+            target="_blank"
+            rel="noreferrer"
```

Comment:
```suggestion
```

Reply:
No strong pref here. Mainly a habit.

Reply:
That way bedrock colleagues would see it in their stats.

Reply:
Never mind, ESLint adds it back, I reverted my change using force-push.